src/main/flight/pid.h

```diff
@@ -160,6 +161,10 @@ typedef struct pidProfile_s {
     uint8_t use_integrated_yaw;             // Selects whether the yaw pidsum should integrated
     uint8_t integrated_yaw_relax;           // Specifies how much integrated yaw should be reduced to offset the drag based yaw component
     uint8_t thrustLinearization;            // Compensation factor for pid linearization
+    uint8_t dterm_cut_percent;              // Amount to cut D by with no gyro activity, zero disables, 20 means cut 20%, 50 means cut 50%
```

Comment:
These changes require the parameter group id to be incremented in `pid.c`, as this is an ARRAY type parameter group. (see https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md.)

Reply:
thanks! done.